locale.html

```diff
@@ -85,7 +85,7 @@ <h1><ins>TimeZonesOfLocale ( _loc_ )</ins></h1>
         1. <ins>Let _locale_ be _loc_.[[Locale]].</ins>
         1. <ins>Assert: _locale_ matches the `unicode_locale_id` production.</ins>
         1. <ins>Let _region_ be the substring of _locale_ corresponding to the `unicode_region_subtag` production of the `unicode_language_id`.</ins>
-        1. <ins>Let _list_ be a List of 1 or more time zone identifiers, which must be String values indicating a Zone or Link name of the IANA Time Zone Database, sorted in descending preference of those in common use in _region_.</ins>
+        1. <ins>Let _list_ be a List of 1 or more unique time zone identifiers, which must be String values indicating a Zone or Link name of the IANA Time Zone Database, sorted in simple lexicographic order, of those in common use in _region_.</ins>
```

Comment:
I assume "simple lexicographic order" is a keyword used elsewhere in the spec?

Reply:
@sffc see Note 2 on https://tc39.es/ecma262/#sec-abstract-relational-comparison.

Reply:
"simple lexicographic order" is only mentioned in a note, so that's not really a normative description, if we want to be strict about this here. Alternatively we could use the same approach as taken in [ModuleNamespaceCreate](https://tc39.es/ecma262/#sec-modulenamespacecreate), which refers to the built-in `%Array.prototype.sort%` function to sort strings.

Reply:
I like this, Change the PR to follow @anba's suggestion.